actions/parse-push/src/run.ts

```diff
@@ -160,19 +139,11 @@ async function getNewCases(commitSha: string): Promise<string[]> {
 }
 
 async function parseCommit(commit: RemoteCommit): Promise<Commit | null> {
-    const slug = await getSlug(commit)
-    const pull_request_html_url = await getPullRequestHtmlUrl(commit.id)
-
-    if (!pull_request_html_url) {
-        return null
-    }
-
+    const slugs = await getSlugs(commit.modified)
     const new_cases = await getNewCases(commit.id)
```

Comment:
Should this be fully async instead of waiting for them in order?

Reply:
It will be fast if we wait on both.